Код для ревью Surf.txt

```diff
@@ -0,0 +1,103 @@
+package ru.surf;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.stream.Stream;
+
+
+public class Index {
+    TreeMap<String, List<Pointer>> invertedIndex;
+
+    ExecutorService pool;
+
+    public Index(ExecutorService pool) {
+        this.pool = pool;
+        invertedIndex = new TreeMap<>();
+    }
+
+    public void indexAllTxtInPath(String pathToDir) throws IOException {
+        Path of = Path.of(pathToDir);
+
+        BlockingQueue<Path> files = new ArrayBlockingQueue<>(2);
+
+        try (Stream<Path> stream = Files.list(of)) {
+            stream.forEach(files::add);
+        }
+
+        pool.submit(new IndexTask(files));
+        pool.submit(new IndexTask(files));
+        pool.submit(new IndexTask(files));
+    }
+
+    public TreeMap<String, List<Pointer>> getInvertedIndex() {
```

Comment:
better move all getters to the single place relatively other methods: I would move them to the place after the constructor